### PR TITLE
feature 36374 - Created the Objecttypen api plugin

### PR DESCRIPTION
**Pull request by Rick-Ritense**
This can't be tested until the object token authentication plugin has been created

### CODE REVIEW

File: projects/valtimo/plugin/src/lib/plugins/objecttypen-api/components/objecttypen-api-configuration/objecttypen-api-configuration.component.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<!--
```
Comment (mritense) [inline projects/valtimo/plugin/src/lib/plugins/objecttypen-api/components/objecttypen-api-configuration/objecttypen-api-configuration.component.html:1]: File should be formatted with prettier

File: projects/valtimo/plugin/src/lib/plugins/objecttypen-api/objecttypen-api-plugin.specification.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+const objecttypenApiPluginSpecification: PluginSpecification = {
+  pluginId: 'objecttypenapi',
+  pluginConfigurationComponent: ObjecttypenApiConfigurationComponent,
+  pluginLogoBase64: OBJECTTYPEN_API_PLUGIN_LOGO_BASE64,
+
+  pluginTranslations: {
+    nl: {
+      title: 'Objecttypen API',
+      url: 'Objecttypen API URL',
+      urlTooltip: 'In dit veld moet de verwijzing komen naar de REST api van Objecttypen',
```
Comment (mritense) [inline projects/valtimo/plugin/src/lib/plugins/objecttypen-api/objecttypen-api-plugin.specification.ts:30]: *API

File: projects/valtimo/plugin/src/lib/plugins/objecttypen-api/objecttypen-api-plugin.specification.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        'Met de Overige Objecten Registratie API-specificaties (Objecttypen) kunnen gemeenten eenduidig objecten (zoals prullenbakken, laadpalen etc.) registreren, opslaan en ontsluiten.',
+      configurationTitle: 'Configuratienaam',
+      configurationTitleTooltip:
+        'Onder deze naam zal de plugin te herkennen zijn in de rest van de applicatie',
+      authenticationPluginConfiguration: 'Configuratie authenticatie-plug-in',
+    },
+    en: {
+      title: 'Objecttypen API',
+      url: 'Objecttypen API URL',
+      urlTooltip: 'This field must contain the URL to the rest API of Objecttypen',
```
Comment (mritense) [inline projects/valtimo/plugin/src/lib/plugins/objecttypen-api/objecttypen-api-plugin.specification.ts:41]: *REST

File: projects/valtimo/plugin/src/lib/plugins/objecttypen-api/objecttypen-api-plugin.specification.ts [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        'With the Other Objects Registration API specifications (Object types), municipalities can unambiguously register, store and access objects (such as trash cans, charging stations, etc.).',
+      configurationTitle: 'Configuration name',
+      configurationTitleTooltip:
+        'With this name the plugin will be recognizable in the rest of the application',
+      authenticationPluginConfiguration: 'Authentication plugin configuration',
+    },
+    de: {
+      title: 'Objecttypen API',
+      url: 'Objecttypen API URL',
+      urlTooltip: 'Dieses Feld muss die URL zur rest API von Objecttypen enthalten',
```
Comment (mritense) [inline projects/valtimo/plugin/src/lib/plugins/objecttypen-api/objecttypen-api-plugin.specification.ts:52]: *REST

File: projects/valtimo/plugin/src/lib/plugins/objecttypen-api/assets/objecttypen-api-plugin-logo.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
...wiKhoMWERUNBiwiKhoMGARUdFgwCKiolHoAasniChJBT2ZPKmAFXQFOwMWUbL6hzg29vWVSQWsoFkwdwQRJSnMNbcCMUsqYH0W8LhBIKIkDQl4XCKpuZMKWO8HPE5zn/cBESVlWMDj3kUCkgpYCwMepz/fviCiJGj/1V4Bj30NCUgqYIVJIDQRRJSEo0Ic+yZKyGD421MFKc3gaCFREmYj+HX6A5SYDxD8ZJwFIoqTdr3YEKXkBsg0nWPQk7ESwbYJI6LOadfQiwh+fZZUc7CDbtceJoI/DaZ0JorDVQh3bV6BEqTBRvujwpyYO8GgRRSlMxHumtQyGCUqbCTXMg1sHhKFVSHlOoS/HmejhA2Uohtdhj1Jy6ScAmabIAriYCmvIvx1qOUglDht79qIymIpZ4Oz4Yk6o10p35TyJKK7/maiG9A5Va8jupOmpQ3+G3G+lP3BeVtE2vrQvqUTpdwlZTmiveZakIe+K4P82E/KHMTbpNOh1rfh7/L7EfzFmVOkzAVRablUSi8p28Hv3/0y/HW4VTF+z59JuR7dyIWINuLnUo4HUenRG3KS19ED6Kb9x7eBAYsorCQD1kvwa3PdknYE6twqBiyi4JIKWI3wR/q7Na1a3gAGLKKgkghYz4Gj8Zs4WcoaMGARdVXcAet2xNuBX7SGIroJbQxY1F3EFbA06cCxoKy0iajrm8KuO2TAou4i6oCVhl+r6g/KWT8plyPawMWARaUoqoClE7D/KmU4KDBtO58q5XkwYBFtSdiApZvE6MDXV0CR2hX+hFNdw9QKBiwiFSRg6TpcnQc5GkU0CTRfS3OioDWvA+Ev86mRUgt/bVNFlmO+D7/KS1RKNGBlS7mk6cnnS2mCvzStLvNvRaeYA9aW6J3iS5myM/w3sW/mv+0AriWk0qRrCbXWtBb+mllNv6TNvA8yZTWIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiIiAoT8H8LwFos5yAGTwAAAABJRU5ErkJggg==';
```
Comment (mritense) [inline projects/valtimo/plugin/src/lib/plugins/objecttypen-api/assets/objecttypen-api-plugin-logo.ts:18]: I noticed this image is square with transparency at the top and bottom. Could you crop out this transparency (so there's no empty space in the logo image) and resize it to a maximum height of 60px as mentioned at https://valtimo.gitbook.io/valtimo-core-docs/extending-valtimo/plugins/custom-plugin-definition#plugin-logo

File: projects/valtimo/plugin/src/lib/plugins/objecttypen-api/components/objecttypen-api-configuration/objecttypen-api-configuration.component.html (hunk 1 above)
Comment (Rick-Ritense) [reply projects/valtimo/plugin/src/lib/plugins/objecttypen-api/components/objecttypen-api-configuration/objecttypen-api-configuration.component.html:1]: Weird if it is incorrect. I have enabled a on reformat code action and on save for prettier in intellij

**Review by mritense — APPROVED** [review]

**Pull request merged**